rust/connlib/tunnel/src/device_channel/tun_linux.rs

```diff
@@ -350,18 +346,95 @@ impl ioctl::Request<SetTunFlagsPayload> {
     }
 }
 
+/// Back up `/etc/resolve.conf` and then modify it in-place
+async fn configure_resolv_conf(
+    dns_config: &[IpAddr],
+    resolv_path: &Path,
+    backup_path: &Path,
+) -> Result<()> {
+    if dns_config.is_empty() {
+        tracing::info!("`dns_config` is empty, leaving `/etc/resolv.conf` unchanged");
```

Comment:
```suggestion
        tracing::debug!("`dns_config` is empty, leaving `/etc/resolv.conf` unchanged");
```

My rule of thumb is: `info!` is reserved for (significant) state changes.